### PR TITLE
adding annotate_base function

### PR DESCRIPTION
**Pull request by john-harrold**
I built this locally and it seems to work. I put the function into read_pptx.R. It seems like the logical location. 

Just to be up front, I don't really have any R experience in terms of developing packages.

### CODE REVIEW

File: R/read_pptx.R [hunk 1]
```diff
@@ -240,6 +240,65 @@ layout_properties <- function( x, layout = NULL, master = NULL ){
   data
 }
 
+#' @export
+#' @title annotate PowerPoint base document
+#' @description generates a slide from each layout in the base document an identifies the layout, master and indexes of each body element.
+#' @param path path to the pptx file to use as base document or NULL to use the officer default
```
Comment (davidgohel) [inline R/read_pptx.R]: Could you document also `output_file`?

File: R/read_pptx.R [hunk 2]
```diff
@@ -240,6 +240,65 @@ layout_properties <- function( x, layout = NULL, master = NULL ){
...
+#' @examples
+#' To generate an anotation of the default base documet with officer and place the output in the file annotated_layout.pptx:
```
Comment (davidgohel) [inline R/read_pptx.R]: This should be commented, it generates an error when *checked*

File: R/read_pptx.R [hunk 3]
```diff
@@ -240,6 +240,65 @@ layout_properties <- function( x, layout = NULL, master = NULL ){
...
+#'   annotate_base()
+#'
+#' To generate an annotation of the base document 'mydoc.pptx' and place the annotated output in 'mydoc_annotate.pptx'
```
Comment (davidgohel) [inline R/read_pptx.R]: This should be commented, it generates an error when *checked*

File: R/read_pptx.R (hunk 2 above)
Comment (john-harrold) [reply R/read_pptx.R]: I'm confused, is it not already commented? I simply copied one of the other chunks of code and modified them.

File: R/read_pptx.R (hunk 1 above)
Comment (john-harrold) [reply R/read_pptx.R]: Sure thing and I'll also document the data that is returned as well.

File: R/read_pptx.R (hunk 2 above)
Comment (john-harrold) [reply R/read_pptx.R]: Oh I see. The descriptive text after @examples needs to be commented.  Got it.

**Comment by john-harrold** [comment]
Ok I think I fixed it in the last commit (13a7644 up there). Do I create a new pull request or can I have it rerun the tests with the most current version?

**Pull request closed without merging**